app/frontend/helpers/printAda.ts

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line no-unused-vars
 import {Lovelace} from '../types'
 export default (coins: Lovelace, decimals = 6): string => {
```

Comment:
I stumbled upon this "decimals" parameter and I think it's kind of misleading as it suggests ADA can have a different amount of decimals than 6, but it is actually meant to be the "precision" to use to display ADA so I'd consider renaming the parameter to "precision" to avoid future confusion

Reply:
Refactored